taxonomies/ingredients.txt

```diff
@@ -19947,6 +19949,31 @@ wikipedia:en:https://en.wikipedia.org/wiki/Amaretto
 #ingredient/amaretto has 15 products in 4 languages @2018-12-21
 
 
+<en:liqueur
+ca:Limoncello
+cs:Limoncello
+de:Limoncello
+en:Limoncello
```

Comment:
```suggestion
en:Limoncello
ca:Limoncello
cs:Limoncello
de:Limoncello
```

Reply:
Looks like `xx:Limoncello` is more appropriate.

Reply:
So it would be:
```
en:Limoncello
ja:リモンチェッロ
ko:리몬첼로
lt:Limončelas
ru:Лимончелло
zh:檸檬酒
xx:Limoncello
wikipedia:en:https://en.wikipedia.org/wiki/Limoncello
wikidata:en:Q736208
# https://world.openfoodfacts.org/cgi/search.pl?search_terms=limoncello&search_simple=1&action=process
# 226 "Limoncello" via OFF search on 2021-01-11
```
right?

Also I don't find it as a category, should I add it the same as in ingredients?

Reply:
Looks good. We should add it as a category as well indeed